stdlib/2and3/traceback.pyi

```diff
@@ -35,7 +35,10 @@ if sys.version_info >= (3, 5):
     def extract_tb(tb: Optional[TracebackType], limit: Optional[int] = ...) -> StackSummary: ...
     def extract_stack(f: Optional[FrameType] = ...,
                       limit: Optional[int] = ...) -> StackSummary: ...
-    def format_list(extracted_list: List[FrameSummary]) -> List[str]: ...
+    # NOTE(https://bugs.python.org/issue34648): Authors who are diligent
+    # enough to type-check their code are authors who are diligent enough to
+    # upgrade their Tuple[str, int, str, Optional[str]]s to FrameSummarys.
```

Comment:
I don't think this kind of rhetoric belongs in a comment. I would be willing to make this accept an Iterable instead of a List, but there's no need for the comments.